app/src/main/java/com/github/onlynotesswent/ui/overview/Overview.kt

```diff
@@ -228,14 +228,12 @@ fun OverviewScreenGrid(
     navigationActions: NavigationActions
 ) {
   CustomLazyGrid(
-      modifier =
-          Modifier.fillMaxSize()
-              .padding(top = 20.dp, bottom = paddingValues.calculateBottomPadding()),
```

Comment:
Nice deletion, since before we had some padding at the bottom of the grid which was showing on top of the grid elements